pulser-core/pulser/backend/noise_model.py

```diff
@@ -44,12 +44,16 @@ class NoiseModel:
             options:
 
             - "dephasing": Random phase (Z) flip (parametrized
-              by `dephasing_rate`).
+              by `dephasing_rate`), commonly characterized experimentally
+              by the T2 time.
             - "relaxation": Noise due to a decay from the Rydberg to
-              the ground state (parametrized by `relaxation_rate`).
+              the ground state (parametrized by `relaxation_rate`), commonly
+              characterized experimentally by the T1 time.
```

Comment:
Nit: perhaps switch the order of dephasing and relaxation, it makes more sense to have T1 and then T2 ?

Reply:
Sure, why not?